great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -200,7 +216,70 @@ def get_metrics(
             parameters=parameters,
         )
 
-        # Obtain enforce_numeric_metric from rule state (i.e., variables and parameters); from instance variable otherwise.
+        metric_values: Union[List[Any], np.ndarray] = []
+
+        metric_value: Union[Any, List[Any]]
+        batch_id: str
+        for batch_id in batch_ids:
+            metric_domain_kwargs["batch_id"] = batch_id
+            metric_configuration_arguments: Dict[str, Any] = {
+                "metric_name": metric_name,
+                "metric_domain_kwargs": metric_domain_kwargs,
+                "metric_value_kwargs": metric_value_kwargs,
+                "metric_dependencies": None,
+            }
+            metric_value = validator.get_metric(
+                metric=MetricConfiguration(**metric_configuration_arguments)
+            )
```

Comment:
Instead of creating a dict and passing it with **kwargs, why not just directly instantiate the MetricConfiguration object with args?

I think this step just adds an unnecessary creation of a dict right?

Reply:
@cdkini Nice idea -- DONE (in next commit).